samples/nrf9160/modem_shell/src/location/location_srv_ext_nrf_cloud.c

```diff
@@ -222,38 +222,26 @@ void location_srv_ext_cellular_handle(
 	if (cloud_resp_enabled) {
 		result = &nrf_cloud_location;
 	}
-	err = location_srv_ext_nrf_cloud_pos_get(cell_info, NULL, result);
-	if (err) {
-		mosh_error("Failed to send cellular location request, err: %d", err);
-		location_cellular_ext_result_set(LOCATION_EXT_RESULT_ERROR, NULL);
-	} else if (cloud_resp_enabled) {
-		location_cellular_ext_result_set(LOCATION_EXT_RESULT_SUCCESS, &nrf_cloud_location);
-	} else {
-		location_cellular_ext_result_set(LOCATION_EXT_RESULT_UNKNOWN, NULL);
-	}
-}
-#endif /* CONFIG_LOCATION_METHOD_CELLULAR */
-
+	err = location_srv_ext_nrf_cloud_location_get(
```

Comment:
I still want to verify tomorrow at the office that this function works with ~20-25 WiFi APs when cellular is also used. There were likely stackoverflows at some point.

Reply:
This works fine. There are some unrelated issues when 25-30 WiFi's are seen.